Dockerfile.shipper

```diff
@@ -1,5 +1,5 @@
 FROM alpine:3.8
 LABEL authors="Parham Doustdar <parham.doustdar@booking.com>, Alexey Surikov <alexey.surikov@booking.com>, Igor Sutton <igor.sutton@booking.com>, Ben Tyler <benjamin.tyler@booking.com>"
```

Comment:
@juliogreff I believe it makes sense to put your name in the list of authors and you are introducing some breaking changes to it.